program/src/queue.rs

```diff
@@ -229,7 +229,8 @@ pub struct FillEvent {
     pub maker_slot: u8,
     pub maker_out: bool, // true if maker order quantity == 0
     pub version: u8,
-    pub padding: [u8; 3],
+    pub market_fees_applied: bool,
+    pub padding: [u8; 2],
```

Comment:
(side note: padding is a misleading name, reserve is better, this is not really padding to align, anyway we I will try to go with reserve naming in v4 for zero copy)